googleapiclient/discovery.py

```diff
@@ -204,8 +204,9 @@ def build(
     cache_discovery: Boolean, whether or not to cache the discovery doc.
     cache: googleapiclient.discovery_cache.base.CacheBase, an optional
       cache object for the discovery documents.
-    client_options: Dictionary or google.api_core.client_options, Client options to set user
-      options on the client. API endpoint should be set through client_options.
+    client_options: Mapping object or google.api_core.client_options, Client
+      options to set user options on the client. API endpoint should be set
```

Comment:
`client` should be written in the same case, upper or lower. Also, `The API endpoint...`